ui/anduril/channel-modes.c

```diff
@@ -135,6 +134,7 @@ uint8_t channel_mode_state(Event event, uint16_t arg) {
         return EVENT_HANDLED;
     }
     #endif
+    #endif  // ifdef USE_CHANNEL_MODE_ARGS
```

Comment:
I'm not sure moving this endif is the correct fix, it implies that NUM_CHANNEL_MODES depends on USE_CHANNEL_MODE_ARGS. See https://github.com/ToyKeeper/anduril/pull/6 for another approach.

Reply:
> I'm not sure moving this endif is the correct fix

Yeah, I was just looking at that and thinking the same thing.  This patch makes the compile work, but it breaks a feature in the process.

Reply:
Interesting, I didn't notice any breakage here despite having used it daily in all my Anduril lights for almost 4 months. Perhaps because I don't use that other feature (which one?) that got broken?

Anyway, glad a better solution was offered and accepted, and thanks @johnou for the review.